[2024-10-06T19:09:28.435+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-06T19:09:33.098+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T02:05:06+07:00 [queued]>
[2024-10-06T19:09:33.290+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T02:05:06+07:00 [queued]>
[2024-10-06T19:09:33.315+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-06T19:09:33.450+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_to_sqlite> on 2024-10-06 19:05:06+00:00
[2024-10-06T19:09:33.481+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-06T19:09:33.488+0000] {standard_task_runner.py:72} INFO - Started process 274 to run task
[2024-10-06T19:09:33.504+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_to_sqlite', 'manual__2024-10-07T02:05:06+07:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp1rrwoz8f']
[2024-10-06T19:09:33.528+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask load_to_sqlite
[2024-10-06T19:09:33.922+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_to_sqlite manual__2024-10-07T02:05:06+07:00 [running]> on host c9ff1ae97c7a
[2024-10-06T19:09:34.556+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2024-10-06T19:05:06+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-07T02:05:06+07:00'
[2024-10-06T19:09:34.566+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-06T19:09:35.691+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-06T19:09:35.786+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-06T19:09:35.788+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_to_sqlite, run_id=manual__2024-10-07T02:05:06+07:00, execution_date=20241006T190506, start_date=20241006T190933, end_date=20241006T190935
[2024-10-06T19:09:36.163+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-06T19:09:36.812+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-06T19:09:36.879+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
